tests/statecheck/consensus_test.go

```diff
@@ -317,15 +316,12 @@ func validateNodes(t *testing.T, genesis *registryAPI.Genesis, source consensusA
 		}
 
 		vrfPubkey := ""
-		if n.VRF != nil {
-			vrfPubkey = n.VRF.ID.String()
-		}
+		vrfPubkey = n.VRF.ID.String()
```

Comment:
Can now remove the above line and do
```suggestion
		vrfPubkey := n.VRF.ID.String()
```

Reply:
:+1:  We can even inline `vrfPubkey` now. Fixed.